src/shared/helpers/urlHelper.ts

```diff
@@ -22,6 +22,7 @@ export const speciesSelector = () => '/species-selector';
 type BrowserUrlParams = {
   genomeId?: string | null;
   focus?: string | null;
+  variantId?: string | null;
```

Comment:
This is not how we are creating links for genome browser. If I am reading the body of the `genomeBrowserVariant` function below, it won't even create a valid genome browser link. In the genome browser, variant is a focus object; and thus, it is passed to the helper as a `focus` parameter. The helper function that generates urls for the genome browser does not distinguish between different types of focus objects. The focus object id passed to it will have to be ready to be used in genome browser urls; and thus will have to have the word "variant" in it.

You can take the `entityIdForUrl` variable provided by `useEntityViewerIds` hook and pass that as a focus object.